v2.0/stop-a-node.md

```diff
@@ -74,7 +74,7 @@ If you need to troubleshoot this command's behavior, you can change its [logging
 
 2. If the node is running in the background and you are using a process manager for automatic restarts, use the process manager to stop the `cockroach` process without restarting it.
 
-    If the node is running in the background and you are not using a process manager, send a kill signal to the `cockroach` process, for example:
+    If the node is running in the background and you are not using a process manager, send a stop signal to the `cockroach` process, for example:
```

Comment:
In this case, since `pkill` is referenced, I'm not sure if there may be a technical distinction between a "kill" signal and some other signal. I'm raising this issue because there has been a discussion about the use of SIGTERM ("terminate") in relation to SIGKILL ("kill"). So we may want to be careful when referring explicitly to a type of signal. I'd suggest getting eng feedback on this one point.

Reply:
This is a good point. I'll revert for these changes for "kill signal" (so I'm not blocking the rest of the changes from being merged) and I'll do some more research on it